doc/source/how-to/documenting.rst

```diff
@@ -506,7 +506,7 @@ Follow these steps to enable multi-version documentation in your project:
 
 - Create a ``gh-pages`` branch in the repository of your project.
 
-- Create a ``release/`` directory and a ``versions.json`` containing:
+- Create a ``versions.json`` in the ``release/`` directory. The content of the ``versions.json`` file should be:
```

Comment:
Since the `release/` directory also needs to be created, maybe "Create a ``release/`` directory, _containing_ a ``versions.json`` file" would be more appropriate?

Reply:
I agree. Let's got with that approach.